index.html

```diff
@@ -293,62 +293,70 @@
 reminder of their humanity. When we use the term [=user=], it is to talk about the specific [=person=]
 who happens to be using a given system at that time.
 
-We define <dfn data-lt="data">personal data</dfn> as any information relating to a [=person=] such
-that:
-
-* this [=person=] is identified, directly or indirectly, by reference to an
-  identifier such as a name, email address, an arbitrary identifier or
-  identification number, an online identifier such as an IP address or any
-  identifier attached to a device this [=person=] may be using, phone number,
-  location data, or factors specific to the physical, physiological, genetic,
-  mental, economic, cultural, or social identity of that [=person=], as well as
-  identifiers derived from such data, for instance through hashing; or
-* this [=person=] could reasonably be reidentified from a conjunction of this
-  data with other data; or
-* the data pertains to a group of people such that a [=person=] may find
-  themselves to be the subject of a treatment related to this group, even if the
-  entity carrying out the treatment has no way to identify that [=person=].
+We define <dfn data-lt="data">personal data</dfn> as any information relating to a [=person=],
+identified or identifiable, directly or indirectly, in particular by reference to an [=identifier=].
+
+[=Data=] is personal if this [=person=] could reasonably be reidentified from a conjunction of this
+data with other data.
 
 Data is <dfn>permanently de-identified</dfn> when there exists a high level of confidence
-that no human subject of the data can be identified, directly or indirectly (e.g., via
-association with an identifier, user agent, or device), by that data alone or in
-combination with other retained or available information, including as being part of a
-group. Note that further considerations relating to groups are covered in the
+that no [=person=] described by the data can be identified, directly or indirectly
+(e.g. via association with an [=identifier=], user agent, or device), by that data alone or in
+combination with other available information, including as being part of a group. Note
+that further considerations relating to groups are covered in the
 <a href="#collective">Collective Issues in Privacy</a> section.
 
-Data is <dfn data-lt="pseudonymous data|pseudonymity">pseudonymous</dfn> when:
+Data is <dfn data-lt="pseudonymous data|pseudonymity">pseudonymous</dfn> when there are
```

Comment:
I think we should break the People & Data section into subsections, so that people can find the "personal data", "permanently de-identified", "pseudonymous data", and "vulnerable person" definitions in the ToC.